index.html

```diff
@@ -1337,7 +1431,34 @@ <h4>Verify Derived Proof (bbs-2023)</h4>
       </section>
 
     </section>
-
+    <section class="informative">
+      <h2>Optional Features</h2>
+      <p>
+The cryptographic properties of BBS signatures permits variants that
+can support advanced functionalities. This specification has limited itself to
+supporting only the most relevant of these enhancements which we explain in the
+following sections. The variables |commitment_with_proof|, |use_pseudonyms|,
+|pid| and |pseudonym| are associated with these features and are not otherwise
+needed for BBS signatures and proofs.
+      </p>
+      <section>
```

Comment:
```suggestion
      <p class="issue at-risk" title="Optional BBS features are at risk">
The optional BBS features described in this section, and included in the 
algorithms in this specification, are at risk and will be removed before
the finalization of this specification if their respective specifications 
at the IETF do not reach RFC status on the same timeline or if there 
are not at least two independent implementations for each optional
feature.
      </p> 
      <section>
```

Reply:
Thanks for the text @msporny. I was planning on filling out the descriptions of the optional features in the sections that I outline. At the same time should I add additional "at risk markers" directly in the various algorithms?